03-lesson-plan.Rmd

```diff
@@ -1,3 +1,11 @@
 # Lesson Plan
 
-Coming soon!
+| Activity | Undergraduate course | Graduate course | |
+| :-- | :-- | :-- | :--|
+| Getting started _Logging into AnVIL_, _Provisioning Galaxy workspace_ | 10-15 minutes | 10-15 minutes | _Provisioning will take 8-10 minutes of waiting_ |
```

Comment:
I like this! Can we add links?

Reply:
I added links to this fork. And yes, that makes sense to create a branch on jhudsl in the future - I wasn't really sure what I was doing when I started this one (:

Reply:
All good!

A few things:

- For links, there shouldn't be a space between [] and ()
- Can we use relative paths? instead of `[_Logging into AnVIL_](https://jhudatascience.org/GDSCN_Book_SARS_Galaxy_on_AnVIL/student-activity-guide.html#set-up)` can we do `[_Logging into AnVIL_](#set-up)` ?
- If you want to preview locally, you can knit the file or you can do `bookdown::serve_book()` - just be sure not to commit any of the rendered files (.md, .html, etc), because the github actions do those for us.